bin/build-modules.js

```diff
@@ -31,15 +31,9 @@ function buildPackage(pkg) {
 }
 
 readDir('packages/node_modules').then(function (packages) {
-  // `pouchdb-for-coverage` has to be built last because it depends on
-  // `pouchdb`, which has a particular index.es.js file as its jsnext:main
-  return Promise.all(packages.filter(function (pkg) {
-    return pkg !== 'pouchdb-for-coverage';
-  }).map(buildPackage)).then(function () {
-    return buildPackage('pouchdb-for-coverage');
+  return Promise.all(packages.map(buildPackage)).catch(function (err) {
+    console.error('build error');
+    console.error(err.stack);
+    process.exit(1);
```

Comment:
There was no need to do these in any particular order anymore, since all the aggressively-bundled packages read from the `src` of another package. This has the additional benefit of faster builds.